test/osml/model_endpoint/me_container.test.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2024 Amazon.com, Inc. or its affiliates.
+ */
+import { App, Stack } from "aws-cdk-lib";
+
+import { MEContainer, OSMLAccount, OSMLVpc } from "../../../lib";
+
+describe("MEContainer", () => {
+  let app: App;
+  let account: OSMLAccount;
+  let stack: Stack;
+  let osmlVpc: OSMLVpc;
+
+  let container: MEContainer;
+
+  beforeEach(() => {
+    app = new App();
+    stack = new Stack(app, "MEContainerStack");
+
+    account = {
```

Comment:
Can we create and vend this test account template to all the tests in another file to deduplicate the code?

Reply:
Good idea! I put this into an own class now.